README.md

```
@@ -1,2 +1,49 @@
+This file is part of @cldn/components.
+
+@cldn/components is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General
+Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any
+later version.
+
+@cldn/components is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+details.
+
+You should have received a copy of the GNU Lesser General Public License along with @cldn/components. If not,
+see https://www.gnu.org/licenses/.
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)